src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
```

Comment:
For the future, better not to use such short names, better to name it `classnames` here